install/install.py

```diff
@@ -103,6 +103,22 @@ def install_restricted_from_cfg(config_file):
         install_in_directory(os.path.expanduser(cp.get(section, "directory")), cp.get(section, "command"))
 
 
+def is_compatible():
+        compatible_value = os.system("which apt-get >> /dev/null")
```

Comment:
If the distro is not Debian based, it outputs an error on the screen (from `which`)

```
which: no apt-get in $PATH
```
which looks like there was an error in OWTF installer.

Instead suppress that error using `2>/dev/null` or `2>&1`.